build/ci/get_operator_github_branch.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -xe
+set +o pipefail
+
+operator_branch=develop
+DOCKER_HUB_USERNAME=csiblock1
+DOCKER_HUB_PASSWORD=$csiblock_dockerhub_password
+wanted_image_tag=`echo $CI_ACTION_REF_NAME | sed 's|/|.|g'`
+
+does_the_docker_image_has_tag(){
+  driver_component=$1
+  does_docker_image_has_tag=false
+  export image_tags=`docker-hub tags --orgname csiblock1 --reponame ibm-block-csi-$driver_component --all-pages | grep $wanted_image_tag | awk '{print$2}'`
+  for tag in $image_tags
+  do
+    if [[ "$tag" == "$wanted_image_tag" ]]; then
+      does_docker_image_has_tag=true
```

Comment:
We can `break` after setting the variable to `true`, no?

Reply:
done